classes/app.rb

```diff
@@ -142,4 +148,21 @@ def retrieve_music
   def retrieve_genre
     retrieve_data('genre.json', 'genre', @genre)
   end
+
+  def list_all_games
+    puts 'List of Games: '
+    puts 'Are empty. Sorry!!!!' if @games.empty?
+    @games.each_with_index do |game, index|
+      puts "#{index + 1} Multiplayer: #{game.multiplayer},
+      Last played at: #{game.last_played_at},
+      Publish date: #{game.publish_date}"
+    end
+  end
+
+  def list_all_authors
+    puts 'Select the author by number:'
+    @authors.each_with_index do |author, index|
+      puts "#{index + 1}. #{author.first_name} #{author.last_name}"
+    end
+  end
```

Comment:
This code is not functioning as expected. It should fetch the data from `./data/game_data/game.json` and `./data/game_data/author.json` but it's not doing so.

In the console is reported the `./data/game_data/game.json`  file as empty 👇

![console-issue](https://user-images.githubusercontent.com/90119799/234883525-79e29817-6dd7-46ea-ab33-13b498689b15.png)

Meanwhile, there is data in `./data/game_data/game.json` and `./data/game_data/author.json`.

![data-issue](https://user-images.githubusercontent.com/90119799/234883597-949d5eb1-f2f5-4800-8240-6226ffe031ef.png)